packages/components/src/range-control/index.js

```diff
@@ -30,6 +30,7 @@ function RangeControl( {
 	min,
 	max,
 	setState,
+	required = false,
```

Comment:
Minor:

- Does it even need to be specified as a prop? Isn't it enough that we pass through `...props` to the rendered inputs via spread?
   - If we do specify it, does it need to have a default? I don't see any practical impact between the default `false` and the non-defaulted `undefined`.
   - If we choose not to specify it, we should remove it from the documentation as well. It's covered well enough by the existing note "Props not included in this set will be applied to the input elements."

Reply:
I chosen not to specify and removed it from the documentation.